PracticeMode.cs

```diff
@@ -161,12 +161,12 @@ public void StartPracticeMode()
             }
             GetSpawns();
             PrintToAllChat($"Practice mode loaded!");
-            Server.PrintToChatAll($" {ChatColors.Green}Spawns: {ChatColors.Default}.spawn, .ctspawn, .tspawn, .bestspawn, .worstspawn");
```

Comment:
What is your reason for removing `Server.` when `Server.PrintToChatAll` works the same?

Reply:
I just replaced what hasn't been replaced yet.
https://github.com/sniperpl/MatchZy/blob/dev/Utility.cs#L25